caravel/views.py

```diff
@@ -1754,7 +1754,21 @@ def show_traceback(self):
     @expose("/welcome")
     def welcome(self):
         """Personalized welcome page"""
-        return self.render_template('caravel/welcome.html', utils=utils)
+
+        """ get faves """
+        session = db.session()
+        FavStar = models.FavStar  # noqa
+        faves = session.query(FavStar).filter_by(user_id=g.user.get_id()).all()
+
+        """ serialize for client """
+        for index, f in enumerate(faves):
+            faves[index] = f.serialize()
+
+        return self.render_template(
+            'caravel/welcome.html',
+            utils=utils,
+            faves=faves
```

Comment:
faves returns an array with json with `u`s prepended to the name. `[{u'class_name': u'Dashboard', u'user_id': 1, u'obj_id': 2}, {u'class_name': u'Dashboard', u'user_id': 1, u'obj_id': 1}]`

need to figure out how to fix this. do we have a pattern for sending model data to the view as json?